tests/lowering/tensor_manipulation/test_view.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+import torch
+import ttnn
+import torch_ttnn
+
+
+class ViewModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, new_shape):
+        return torch.reshape(x, new_shape)
+
+
+class ViewAfterOpModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, new_shape):
+        abs = torch.abs(x)
+        return torch.reshape(abs, new_shape)
+
+
+class ViewBeforeOpModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, new_shape):
+        reshape = torch.reshape(x, new_shape)
+        return torch.abs(reshape)
+
+
+class ViewBetweenOpModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, x, new_shape):
+        abs = torch.abs(x)
+        reshape = torch.reshape(abs, new_shape)
+        return torch.abs(reshape)
+
+
+@pytest.mark.parametrize(
+    "module",
+    [
+        ViewModule(),
+        ViewAfterOpModule(),
+        ViewBeforeOpModule(),
+        ViewBetweenOpModule(),
+    ],
+)
+@pytest.mark.parametrize(
+    "input_shape, new_shape",
+    [
```

Comment:
The reason for testing with different positions of `reshape` is because of data layout changes (e.g. row major) for some shapes. This is to make sure other lowered ops have no conflicts.